packages/headless/src/composables/menuButton/createMenuButton.ts

```diff
@@ -7,6 +7,7 @@ type CreateMenuButtonOptions = {
   isExpanded: Readonly<Ref<boolean>>;
   trigger: Readonly<Ref<"hover" | "click">>;
   onToggle: () => void;
+  disabled?: Readonly<Ref<boolean, boolean>>;
```

Comment:
```suggestion
  disabled?: Readonly<Ref<boolean>>;
```

Isn't this the same?